src/MultiBridgeMessageReceiver.sol

```diff
@@ -183,11 +183,13 @@ contract MultiBridgeMessageReceiver is IMultiBridgeMessageReceiver, ExecutorAwar
         address[] calldata _receiverAdapters,
         bool[] calldata _operations
     ) external override onlyGlobalOwner {
-        /// @dev updates quorum here
-        _updateQuorum(_newQuorum);
-
-        /// @dev updates receiver adapter here
-        _updateReceiverAdapters(_receiverAdapters, _operations);
+        if (_newQuorum > quorum) {
+            _updateReceiverAdapters(_receiverAdapters, _operations);
```

Comment:
We would have a problem where the order of operations within `_updateReceiverAdapters()` makes a difference. Specifically, a scenario that raises the quorum and simultaneously removes existing adapters and adds new ones might fail if the removals are listed first in the array.
Assume the following scenario:
Current State:
- Quorum: 2
- Adapters: A, B

Update Parameters:
- New quorum: 3
- Adapters: B, C, D
- Operations: false, true, true

This would fail because the first remove operation would violate the existing quorum. What we'd like to do is process all the add operations first before removals.

Reply:
Cleanup the logic a bit. One question remains whether a quorum of 0 is ever valid. PTAL.

Reply:
> Should the constructor take in an initial list of adapters and quorum?
Makes sense. Can we update this, please?

Reply:
Done.